tools/scylla-sstable.cc

```diff
@@ -1056,6 +1059,11 @@ sstring disk_string_to_string(const sstables::disk_string<Integer>& ds) {
     return sstring(ds.value.begin(), ds.value.end());
 }
 
+template <typename Integer>
+dht::token disk_string_to_token(const sstables::disk_string<Integer>& ds) {
```

Comment:
instead of defining a template, how about using a regular function? as we are handling an on-disk format represented by `disk_token_range`, in which, `disk_token_bound::token` is always a `disk_string<uint16_t>`. so probably
```c++
dht::token disk_token_bound_to_token(const sstables::disk_token_bound& bound) {
    return dht::token(dht::token::kind::key, bytes_view(bound.token));
}
```

Reply:
I like this disk_token_bound_to_token(const sstables::disk_token_bound& bound). will change, thanks.